### DISCUSSION TITLE
Doubt regarding to contributing to cuda-Quantum

### DISCUSSION THREAD

**Discussion by Gaurang-Belekar** [NVIDIA/cuda-quantum#1530] [Q&A]
I am facing an issue as

```
Clean up / Clean up GHCR images (cuda-quantum-dev)
Failed in 2 seconds
```

Can someone point out where am I going wrong and how can I test the same.

Thanks

**Comment by poojarao8** [comment]
Since this issue is related to your PR, it is better to discuss it there.
